staging/src/k8s.io/component-base/cli/globalflag/globalflags.go

```diff
@@ -66,3 +68,20 @@ func Register(local *pflag.FlagSet, globalName string) {
 		panic(fmt.Sprintf("failed to find flag in global flagset (flag): %s", globalName))
 	}
 }
+
+// RegisterGlobalPflag adds a flag to local that targets the Value associated with the Flag
+// named globalName in pflag.CommandLine.
+func RegisterGlobalPflag(local *pflag.FlagSet, globalName string) {
+	if f := pflag.CommandLine.Lookup(globalName); f != nil {
+		f.Name = normalize(f.Name)
+		local.AddFlag(f)
+	} else {
+		panic(fmt.Sprintf("failed to find flag in global flagset (pflag): %s", globalName))
+	}
+}
+
+// RegisterDeprecated registers the flag with register, and then marks it deprecated.
+func RegisterDeprecated(local *pflag.FlagSet, globalName, deprecated string) {
```

Comment:
Name the flagset `fs` here and above, instead of `local`, to maintain consistency with the other funcs in this file.

Reply:
fix